polls/templates/polls/chat.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+
+<div class="container">
+<form id="chatForm">
+  <textarea id="userInput"></textarea>
+  <button type="button" onclick="sendMessage()">Send</button>
+</form>
+<label id="responseLabel"></label>
+</div>
+<script>
+  function sendMessage() {
+    const text = document.getElementById('userInput').value;
+    fetch('/chat/', {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({message: text})
+    })
+    .then(response => response.json())
+    .then(data => {
+      document.getElementById('responseLabel').textContent = data.reply;
+    });
+  }
+</script>
+
+{% endblock content %}
```

Comment:
Add some styling. It doesn't looks good. Thanks.

Reply:
done please review

Reply:
Because the action takes time, It will be better to add a loading state in the button. Also there is an issue with the output result formatting. Please apply some CSS ex: `white-space: pre-line;` to make the output formatting better.

Current:
![Screenshot 2024-10-04 at 12 54 23 PM](https://github.com/user-attachments/assets/d56a6d33-3c13-47e2-91fd-28080536bdf7)

Expected:
![Screenshot 2024-10-04 at 1 09 11 PM](https://github.com/user-attachments/assets/bb144f0c-196b-459f-922a-b282e920810f)